common/lib/xmodule/xmodule/js/spec/video/sjson_spec.js

```diff
@@ -30,6 +30,31 @@ function (Sjson) {
             expect(sjson.search(21620)).toEqual(4);
             expect(sjson.search(24920)).toEqual(5);
         });
+
+        it('search returns -1 for a value outside the bounds of the array', function() {
+            expect(sjson.search(10024920)).toEqual(sjson.getCaptions().length - 1);
+        });
+
+        it('search only searches through a subrange of times if start / end times are specified', function () {
+            var results = sjson.filter(6000, 10000);
```

Comment:
What do these numbers represent?

They're used multiple times; should they be constants?